src/psyclone/tests/psyir/transformations/arrayrange2loop_trans_test.py

```diff
@@ -89,7 +90,8 @@ def create_indexed_range(symbol_table):
 
     '''
     lbound = Literal("2", INTEGER_TYPE)
-    ubound = Reference(symbol_table.lookup("n"))
+    symbol = symbol_table.lookup("n")
```

Comment:
For some reason GitHub is not letting me continue the original conversation. What I meant was that it might be better/more obvious to pass `create_stepped_range()` a `Symbol` rather than a `SymbolTable`. However, I take your point that this is only test code so as long as it's clear then it's OK.

Reply:
Ah yes, I understand now. Your suggestion is better so I've changed the code.